target-platform/target-platform.target

```diff
@@ -106,17 +108,32 @@
 				</dependency>
 			</dependencies>
 		</location>
-		<location includeDependencyDepth="none" includeDependencyScopes="compile" includeSource="true" label="Maven-archetype" missingManifest="generate" type="Maven">
+		<location includeDependencyDepth="none" includeDependencyScopes="compile" includeSource="true" label="Maven-archetype Common" missingManifest="generate" type="Maven">
 			<dependencies>
 				<dependency>
 					<groupId>org.apache.maven.archetype</groupId>
-					<artifactId>archetype-catalog</artifactId>
+					<artifactId>archetype-common</artifactId>
 					<version>3.2.1</version>
 					<type>jar</type>
 				</dependency>
+			</dependencies>
+			<instructions><![CDATA[
+Bundle-Name:           M2Eclipse supplement fragment for ${mvnGroupId}:${mvnArtifactId}:${mvnVersion}
+version:               ${version_cleanup;${mvnVersion}}
+Bundle-SymbolicName:   org.eclipse.m2e.archetype.common
+Bundle-Version:        ${version}.1
```

Comment:
this will result in a `3.2.1.1` version is this really desired?

Reply:
Yes it is.
It is a (manual) qualifier bump so that those that use the snapshots don't miss the update.
When there is a new archetype version, it can be removed.

Reply:
1 is really a bad qualifier, why not do like in maven-runtime and use 

`${version}01` what will result in 3.2.101